<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIssueDialog</class>
 <widget class="QDialog" name="AddIssueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="rdb_load_from_redmine">
     <property name="font">
      <font>
       <family>B Yekan</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>از پیگیر</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="led_issue_id"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>B Yekan</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>شماره مسئله:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rdb_new_issue">
     <property name="font">
      <font>
       <family>B Yekan</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>جدید</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="led_subject"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>B Yekan</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>عنوان:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cmb_category"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>B Yekan</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>دسته بندی:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>led_issue_id</tabstop>
  <tabstop>rdb_new_issue</tabstop>
  <tabstop>led_subject</tabstop>
  <tabstop>cmb_category</tabstop>
  <tabstop>rdb_load_from_redmine</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddIssueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddIssueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
